src/accountsdb/fuzz.zig

```diff
@@ -108,12 +108,15 @@ pub fn run(seed: u64, args: *std.process.ArgIterator) !void {
     defer allocator.destroy(exit);
     exit.* = std.atomic.Value(bool).init(false);
 
-    const manager_handle = try std.Thread.spawn(.{}, AccountsDB.runManagerLoop, .{ &accounts_db, AccountsDB.ManagerLoopConfig{
-        .exit = exit,
-        .slots_per_full_snapshot = 50_000,
-        .slots_per_incremental_snapshot = 5_000,
-        .zstd_nb_workers = @intCast(std.Thread.getCpuCount() catch 0),
-    } });
+    const manager_handle = try std.Thread.spawn(.{}, AccountsDB.runManagerLoop, .{
+        &accounts_db, AccountsDB.ManagerLoopConfig{
+            .exit = exit,
+            .slots_per_full_snapshot = 50_000,
+            .slots_per_incremental_snapshot = 5_000,
+            .zstd_nb_workers = 0,
+            // .zstd_nb_workers = @intCast(std.Thread.getCpuCount() catch 0), // TODO: breaks ??
```

Comment:
@InKryption can you check this error out -- doesnt seem to work with any value != 0 ?

```
error: InvalidParameters
/Users/x19/.cache/zig/p/12203115da879b68a6631e2272cd3418280375d4f41acfe9ea48301f8aab535691ec/src/compress.zig:231:75: 0x104d143d7 in init (fuzz)
        if (isError(c.ZSTD_CCtx_setParameter(h, 400, params.nb_workers))) return error.InvalidParameters;
                                                                          ^
/Users/x19/Documents/sig/src/accountsdb/db.zig:1145:33: 0x104ec90bb in runManagerLoop (fuzz)
        const zstd_compressor = try zstd.Compressor.init(.{
```